### ISSUE TITLE
Use `output_loglevel='quiet'` with cmd.run in Jinja

### ISSUE THREAD

**Issue by lorengordon** [plus3it/ash-linux-formula#199]
The salt logs are quite noisy with false errors, when using cmd.run/cmd.shell with grep or test in a jinja block to control execution of a state. Any errors are still logged with a scary looking ERROR message. Would reduce some false issue reporting and make it easier to troubleshoot if those errors were not logged, which can be accomplished by passing `output_loglevel='quiet'` to both cmd.run/cmd.shell.

**Comment by lorengordon** [comment]
clean output:

```
# cat /srv/salt/test.sls
{{ salt.cmd.shell('test -f /bar', output_loglevel='quiet') }}

test:
  cmd.run:
    - name: echo bar

# salt-call --local state.sls test -l warning
local:
----------
          ID: test
    Function: cmd.run
        Name: echo bar
      Result: True
     Comment: Command "echo bar" run
     Started: 01:19:25.513775
    Duration: 3.578 ms
     Changes:
              ----------
              pid:
                  2617
              retcode:
                  0
              stderr:
              stdout:
                  bar

Summary for local
------------
Succeeded: 1 (changed=1)
Failed:    0
------------
Total states run:     1
```

vs silly non-ERROR output

```
# cat /srv/salt/test.sls
{{ salt.cmd.shell('test -f /bar') }}

test:
  cmd.run:
    - name: echo bar

]# salt-call --local state.sls test -l warning
[ERROR   ] Command 'test -f /bar' failed with return code: 1
[ERROR   ] output:
local:
----------
          ID: test
    Function: cmd.run
        Name: echo bar
      Result: True
     Comment: Command "echo bar" run
     Started: 01:19:25.513775
    Duration: 3.578 ms
     Changes:
              ----------
              pid:
                  2617
              retcode:
                  0
              stderr:
              stdout:
                  bar

Summary for local
------------
Succeeded: 1 (changed=1)
Failed:    0
------------
Total states run:     1
```

**Comment by ferricoxide** [comment]
Dammit... That's gonna blow out screen margins.

**Comment by lorengordon** [comment]
Heh. Can multi-line jinja pretty easily.

```
{{
    salt.cmd.shell(
        'test -f /bar',
        output_loglevel='quiet')
}}
```

**Comment by ferricoxide** [comment]
Ok, not seeing much of the enumerated construct. I see plenty of ...`*if not salt['cmd.shell']('grep`... constructs:
~~~
$ find ash-linux/el6 -name "*.sls" | xargs grep -El "if.*cmd\."
ash-linux/el6/SCAPonly/low/CCE-26610-6.sls
ash-linux/el6/SCAPonly/low/CCE-26691-6.sls
ash-linux/el6/STIGbyID/cat2/V38518.sls
ash-linux/el6/STIGbyID/cat2/V38519.sls
ash-linux/el6/STIGbyID/cat2/V38623.sls
ash-linux/el6/STIGbyID/cat3/V38530.sls
ash-linux/el6/STIGbyID/cat3/V38540.sls
ash-linux/el6/STIGbyID/cat3/V38541.sls
ash-linux/el6/STIGbyID/cat3/V38543.sls
ash-linux/el6/STIGbyID/cat3/V38545.sls
ash-linux/el6/STIGbyID/cat3/V38547.sls
ash-linux/el6/STIGbyID/cat3/V38550.sls
ash-linux/el6/STIGbyID/cat3/V38552.sls
ash-linux/el6/STIGbyID/cat3/V38554.sls
ash-linux/el6/STIGbyID/cat3/V38556.sls
ash-linux/el6/STIGbyID/cat3/V38557.sls
ash-linux/el6/STIGbyID/cat3/V38558.sls
ash-linux/el6/STIGbyID/cat3/V38559.sls
ash-linux/el6/STIGbyID/cat3/V38561.sls
ash-linux/el6/STIGbyID/cat3/V38563.sls
ash-linux/el6/STIGbyID/cat3/V38565.sls
ash-linux/el6/STIGbyID/cat3/V38566.sls
ash-linux/el6/STIGbyID/cat3/V38568.sls
ash-linux/el6/STIGbyID/cat3/V38575.sls
ash-linux/el6/STIGbyID/cat3/V38578.sls
ash-linux/el6/STIGbyID/cat3/V38635.sls
~~~
And seeing test statements in conditionals (e.g., `- onlyif:`  and/or `- unless` blocks):
~~~
$ find ash-linux/el6 -name "*.sls" | xargs grep -l "'test "
ash-linux/el6/SCAPonly/medium/CCE-26741-9.sls
ash-linux/el6/STIGbyID/cat1/V38497.sls
ash-linux/el6/STIGbyID/cat1/V38666.sls
ash-linux/el6/STIGbyID/cat1/V38668.sls
ash-linux/el6/STIGbyID/cat2/V38490.sls
ash-linux/el6/STIGbyID/cat2/V38501.sls
ash-linux/el6/STIGbyID/cat2/V38524.sls
ash-linux/el6/STIGbyID/cat2/V38526.sls
ash-linux/el6/STIGbyID/cat2/V38532.sls
ash-linux/el6/STIGbyID/cat2/V38542.sls
ash-linux/el6/STIGbyID/cat2/V38546.sls
ash-linux/el6/STIGbyID/cat2/V38573.sls
ash-linux/el6/STIGbyID/cat2/V38574.sls
ash-linux/el6/STIGbyID/cat2/V38592.sls
ash-linux/el6/STIGbyID/cat2/V38600.sls
ash-linux/el6/STIGbyID/cat2/V38658.sls
ash-linux/el6/STIGbyID/cat2/V51875.sls
ash-linux/el6/STIGbyID/cat3/V38482.sls
ash-linux/el6/STIGbyID/cat3/V38535.sls
ash-linux/el6/STIGbyID/cat3/V38537.sls
ash-linux/el6/STIGbyID/cat3/V38569.sls
ash-linux/el6/STIGbyID/cat3/V38570.sls
ash-linux/el6/STIGbyID/cat3/V38571.sls
ash-linux/el6/STIGbyID/cat3/V38572.sls
ash-linux/el6/STIGbyID/cat3/V38655.sls
ash-linux/el6/STIGbyID/cat3/V38693.sls
~~~
Does the issue apply to all of the above or only in the explicit Jinja `if`/`elif`

**Comment by lorengordon** [comment]
Closed by #200
